tsconfig.json

```diff
@@ -12,9 +12,57 @@
     "exactOptionalPropertyTypes": true,
     "jsx": "react-jsx",
     "baseUrl": ".",
+    "composite": true,
+    "incremental": true,
     "paths": {
       "waku": ["./packages/waku/src/main.ts"],
       "waku/*": ["./packages/waku/src/*.ts"]
     }
-  }
+  },
+  "files": [],
+  "include": [],
+  "references": [
+    // Packages
+    {
+      "path": "./packages/waku"
+    },
+    {
+      "path": "./packages/create-waku"
+    },
+    {
+      "path": "./packages/website"
+    },
+    // Examples
```

Comment:
We have 10_ and 11_ now.

Do we need all references??

Reply:
yep